website-next/components/common/env-vars.ts

```diff
@@ -36,7 +36,7 @@ type BackendType =
   | 'direct' // direct calls to the backend
 
 const LOCAL_BACKEND_PORTS: { [type in BackendType]: number } = {
-  direct: 8001,
+  direct: 8000,
```

Comment:
What is with this change?

Reply:
while I was trying this out locally I was hitting some CORS issues with the haskell server, which were solved by hitting the same origin as the editor

Reply:
Riiight ok 👍